yabt/graph_test.py

```diff
@@ -207,61 +213,81 @@ def test_target_graph_intenral_dir(basic_conf):
         set(build_context.target_graph.edges))
 
 
-def test_stable_topological_sort():
-    """Test that my modified topological sort is stable.
-
-    Note - not doing many cycles because I saw that even with the non-stable
-    implementation, it is stable in the context of the same process...
-    (only reruns showed the unstable behavior)
+def subgraph_stable_topsort_test(graph):
+    """Test that sort is stable for subgraph"""
+    def shuffle_graph(graph):
```

Comment:
I don't see a good reason to use create an inner method here.
Move it outside the test method please